commons/go/lib/utils/utils.go

```diff
@@ -373,6 +373,41 @@ func ParseFileNames(files []types.File, testGlobs []string) ([]Node, error) {
 	return nodes, nil
 }
 
+func ParseFileNamesOfKnownLanguage(files []types.File, testGlobs []string, language string) []Node {
+	nodes := make([]Node, 0)
+	for _, file := range files {
+		path := file.Name
+		if len(path) == 0 {
+			continue
+		}
+		var node *Node
+		var err error
+		switch language {
+		case "csharp":
+			node, err = ParseCsharpNode(file, testGlobs)
+			if err != nil {
+				continue
+			}
+		case "java", "scala", "kotlin":
+			node, err = ParseJavaNode(file)
+			if err != nil {
+				continue
+			}
+		case "python":
+			node = ParsePythonNode(file, testGlobs)
+		case "ruby":
+			node = ParseRubyNode(file, testGlobs)
+		default:
+			node, err = ParsePathBasedNodeAndType(file, testGlobs)
+			if err != nil {
+				continue
+			}
+		}
+		nodes = append(nodes, *node)
```

Comment:
Add a single `err` check here.

```suggestion
                if err != nil {
                    continue
                 }
		nodes = append(nodes, *node)
```

Reply:
done